lib/common/events.ts

```diff
@@ -24,6 +29,9 @@ export const ZONE_SYMBOL_PREFIX = '__zone_symbol__';
 
 const EVENT_NAME_SYMBOL_REGX = /^__zone_symbol__(\w+)(true|false)$/;
 
+const _global =
```

Comment:
can we get `global` from `__load_patch`? I would prefer to have the `global` logic centralized.